tpu-provisioner/test/integration/controller/suite_test.go

```diff
@@ -68,14 +76,21 @@ var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
 	By("bootstrapping test environment")
-	testEnv = &envtest.Environment{}
+	testEnv = &envtest.Environment{
+		// Install JobSet CRD which is required for the integration tests.
+		CRDDirectoryPaths:     []string{filepath.Join("..", "..", "crds")},
```

Comment:
Since we vendor the YAML files, I would add a comment to what commands you ran to do so. I would prefer to name the file something like `jobset.v0.5.0.yaml` or something to make the version super visible as well.

Reply:
Updated jobset manifest name to include the version.

Reply:
Updated Makefile to download jobset manifest before running tests.